starlark/eval_test.go

```diff
@@ -339,40 +339,45 @@ f()
 	}
 }
 
-func Benchmark(b *testing.B) {
+func BenchmarkFile(b *testing.B) {
 	testdata := starlarktest.DataFile("starlark", ".")
 	thread := new(starlark.Thread)
+	// TODO(josharian): instead auto-detect all files named bench_*.star
+	// and rename benchmark.star to bench_something.star, possibly
+	// splitting it up in the process.
 	for _, file := range []string{
-		"testdata/benchmark.star",
-		// ...
+		"benchmark.star",
+		"bench_repeat.star",
 	} {
-		filename := filepath.Join(testdata, file)
+		filename := filepath.Join(testdata, "testdata", file)
 
-		// Evaluate the file once.
-		globals, err := starlark.ExecFile(thread, filename, nil, nil)
-		if err != nil {
-			reportEvalError(b, err)
-		}
+		b.Run(file, func(b *testing.B) {
+			// Evaluate the file once.
+			globals, err := starlark.ExecFile(thread, filename, nil, nil)
```

Comment:
I don't consider the ExecFile time relevant to the benchmark, so let's eliminate the outer b.Run.